### DISCUSSION TITLE
Enable Numlock + Remember view choices + Adjust subtitle window (not always on top)

### DISCUSSION THREAD

**Discussion by acid-magic** [URUWorks/TeroSubtitler#187] [Ideas]
I don't know if it was intentional, but it would be great to be able to enter numbers using numeric keypad on right.
When i turn off video preview it turns on when i restart program. One more thing ;), it will be great if adjust subtitle is not always on top so we can use scrollbar and check what is first / last text on subtitle.

**Comment by URUWorks** [comment]
Thanks for the suggestions, we are going to implement everything possible.

Regarding numpad, this is because the video has shortcuts assigned with some number keys, it will probably be changed for the next version.
